pom.xml

```diff
@@ -19,7 +19,7 @@
     <jsonschema_paths>acq-models/mod-orgs/schemas,raml-util/schemas</jsonschema_paths>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${basedir}/raml-util</ramlfiles_util_path>
-    <raml-module-builder.version>25.0.1</raml-module-builder.version>
+    <raml-module-builder.version>28.1.1</raml-module-builder.version>
```

Comment:
any reason to leave it at 28. when we upgraded the others to 29?

Reply:
@KVupp,
for 29.0.1 I catch NPE and wasn't able to easy fix this. If I understand correctly we need to upgrade RMB at least to version with fixed memory utilization issue > 27.1.2 so I decided to upgrade this to 28.1.1 because I worry that this can take much time and affect release. 
NPE:
``2019-12-01T17:23:40,615 INFO  [main                  ] InterfaceCredentialsTest --- mod-organizations-storage InterfaceCredential test: Fetching InterfaceCredential with ID: 14e81009-0f98-45a0-b8e6-e25547beb22f
2019-12-01T17:23:40,619 INFO  [vert.x-worker-thread-10] CQLWrapper CRITERION >>> SQL: WHERE (jsonb->>'interfaceId') = '14e81009-0f98-45a0-b8e6-e25547beb22f'    >>>WHERE (jsonb->>'interfaceId') = '14e81009-0f98-45a0-b8e6-e25547beb22f'
2019-12-01T17:23:40,620 ERROR [vert.x-worker-thread-11] PostgresClient null
java.lang.NullPointerException: null
	at org.folio.rest.persist.PostgresClient.deserializeRow(PostgresClient.java:2443) ~[domain-models-runtime-29.0.1.jar:?]
	at org.folio.rest.persist.PostgresClient.deserializeResults(PostgresClient.java:2384) ~[domain-models-runtime-29.0.1.jar:?]
	at org.folio.rest.persist.PostgresClient.processResults(PostgresClient.java:2352) ~[domain-models-runtime-29.0.1.jar:?]
	at org.folio.rest.persist.PostgresClient.lambda$35(PostgresClient.java:1596) ~[domain-models-runtime-29.0.1.jar:?]
	at org.folio.rest.persist.PostgresClient.lambda$43(PostgresClient.java:1875) ~[domain-models-runtime-29.0.1.jar:?]
	at org.folio.rest.persist.PostgresClient.lambda$57(PostgresClient.java:2585) ~[domain-models-runtime-29.0.1.jar:?]
	at io.vertx.ext.asyncsql.impl.AsyncSQLConnectionImpl.lambda$handleAsyncQueryResultToResultSet$11(AsyncSQLConnectionImpl.java:321) ~[vertx-mysql-postgresql-client-jasync-3.8.1.jar:3.8.1]
	at io.vertx.ext.asyncsql.impl.ConversionUtils.lambda$connectCompletableFutureWithHandler$4(ConversionUtils.java:97) ~[vertx-mysql-postgresql-client-jasync-3.8.1.jar:3.8.1]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:760) ~[?:1.8.0_231]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:736) ~[?:1.8.0_231]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_231]
	at io.vertx.ext.asyncsql.impl.VxContextAwareExecutor.lambda$execute$0(VxContextAwareExecutor.java:22) ~[vertx-mysql-postgresql-client-jasync-3.8.1.jar:3.8.1]
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:369) ~[vertx-core-3.8.1.jar:3.8.1]
	at io.vertx.core.impl.WorkerContext.lambda$wrapTask$0(WorkerContext.java:35) ~[vertx-core-3.8.1.jar:3.8.1]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[vertx-core-3.8.1.jar:3.8.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_231]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_231]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.39.Final.jar:4.1.39.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_231]``

Reply:
ok makes sense.